test/Docfx.MarkdigEngine.Extensions.Tests/QuoteSectionNoteTest.cs

```diff
@@ -275,12 +303,12 @@ public void TestDfmVideo_MixWithNote()
 
     > code text 1-1
     > [!NOTE]
-    > code text 1-2  
+    > code text 1-2
```

Comment:
don't remove these trailing whitespaces, they represent softbreaks in markdown.

Reply:
Mmm, i tried doing that but somehow my editor keeps on removing it.  Will try again

Reply:
This is now fixed